src/NServiceBus.Persistence.DynamoDB.TransactionalSession.AcceptanceTests/Infrastructure/RecoverabilityPolicy.cs

```diff
@@ -0,0 +1,18 @@
+namespace NServiceBus.TransactionalSession.AcceptanceTests;
+
+using System;
+using Persistence.DynamoDB;
+using Transport;
+
+static class RecoverabilityPolicy
+{
+    public static RecoverabilityAction Invoke(RecoverabilityConfig config, ErrorContext errorContext)
```

Comment:
I think even an immediate retry should be good enough to handle these cases. It might also be easier to just enable 1 immediate retry by default for the tx session att?

Reply:
I felt this is more robust but yeah also not against that. I will revert